### PR TITLE
NeuroVault Harvester & Normalizer

### PR DESCRIPTION
**Pull request by caseyrollins**

### CODE REVIEW

**Comment by caseyrollins** [comment]
@chrisseto 🏇

File: providers/org/neurovault/normalizer.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        RunPython('format_doi_url', Maybe(ctx, 'DOI')),
+        Maybe(ctx, 'paper_url')
+    ))
+
+    def parse_names(self, authors):
+        return re.split(',\s|\sand\s', authors)
+
+    def format_doi_url(self, doi):
+        if doi:
+            return format_doi_as_url(self, doi)
```
Comment (chrisseto) [inline providers/org/neurovault/normalizer.py]: Add an explicit `return None`.

Comment (caseyrollins) [inline providers/org/neurovault/normalizer.py]: @chrisseto done 🍰

File: providers/org/neurovault/harvester.py [hunk 2]
```diff
@@ -2,4 +2,15 @@
...
+    url = 'http://www.neurovault.org/api/collections/?format=json'
+
+    def do_harvest(self, start_date, end_date):
+        api_url = self.url
+        while api_url:
+            response = self.requests.get(api_url)
+            records = response.json()
+            for record in records['results']:
+                record_id = record['id']
+                yield(record_id, record)
```
Comment (chrisseto) [inline providers/org/neurovault/harvester.py]: Space between `yield` and `(`

**Comment by chrisseto** [comment]
Could you add the app config and the migration?

**Comment by caseyrollins** [comment]
@chrisseto updates made 🍏  And this provider has an existing app config and migration.

**Pull request merged**